1.9.0

```diff
@@ -258,6 +258,27 @@
           }
         }
       ]
+    },
+    {
+      "sobjectName":"Campaign",
+      "instructionTypeName": "mdTaskListView",
+      "taskList":[
+        {
+          "jiraTicket": "AC-58366",
+          "description": "Create new Campaign Hierarchy View",
+          "actionType": "ListView",
+          "actionTypeName": "createListView",
+          "optionsMap": {
+            "fullName": "Campaign.Org_Campaign_Hierarchy",
```

Comment:
use actionTypeName for this instead of optionsMap (similar to the other PR)

Reply:
done